PortalFetch/crawler_download.py

```diff
@@ -204,6 +206,38 @@ def waitUtilPageLoaded(driver, count):
     raise ElementNotVisibleException("Could not load the full page!")
 
 
+def generateFilenameAndQuarter(quarterValue):
+    """Return quarter and fileName.
+
+    Args:
+        quarterValue:the quarter_value in crawler.config
+    Returns:
+        quarter str and fileName str
+
+    """
+    year = quarterValue[0:4]
+    resultList = []
+    quarterSwitcher = {
+        "1": "Summer",
+        "2": "Fall",
+        "3": "Winter",
+        "4": "Spring",
+    }
+    schoolSwitcher = {
+        "1": "Foothill",
+        "2": "De Anza",
+    }
+    school = schoolSwitcher.get(quarterValue[5], "")
+    quarter = quarterSwitcher.get(quarterValue[4], "")
+    qr = year + " " + quarter + " " + school
+    resultList.append(qr)
+    if school == "De Anza":
+        school = "De_Anza"
+    fn = year + "_" + quarter + "_" + school + "_courseData.json"
```

Comment:
change "fn" to a meaningful name. 

Readings:
[Clean code 101 — Meaningful names and functions](https://medium.com/coding-skills/clean-code-101-meaningful-names-and-functions-bf450456d90c#:~:text=How%20meaningful%20%3F,does%20not%20reveal%20its%20intent.)